### ISSUE TITLE
RVM prompt broken

### ISSUE THREAD

**Issue by HaoZeke** [bhilburn/powerlevel9k#828]
# Details

Hi. I have a rather odd problem.

When I have the following setup in my `.zshrc`:

```bash
POWERLEVEL9K_LEFT_PROMPT_ELEMENTS=(dir newline rbenv virtualenv vcs)
```

I get this
![rbenvbug](https://user-images.githubusercontent.com/4336207/39730310-80565b40-527e-11e8-9598-b5a5d9675ac4.png)

Weirdly though when I have the following:

```bash
POWERLEVEL9K_LEFT_PROMPT_ELEMENTS=(dir newline rvm virtualenv vcs)
```

I get nothing after reloading the shell.

![rvmbug](https://user-images.githubusercontent.com/4336207/39730374-dfc5a306-527e-11e8-8d82-454d29af4aab.png)

# System Details

![sysdetails](https://user-images.githubusercontent.com/4336207/39730396-f8bd0d40-527e-11e8-8a7a-fbed55126967.png)

* My `powerlevel9k` setup is via `zplug`:

``` bash
zplug "bhilburn/powerlevel9k", use:powerlevel9k.zsh-theme, defer:3
```

I am using the [current master](https://github.com/bhilburn/powerlevel9k/commit/13b0e383a066c9c37d5cf71d4af521f9b73a173e), determined via:

```bash
$ git rev-parse HEAD
13b0e383a066c9c37d5cf71d4af521f9b73a173e
```

**Comment by HaoZeke** [comment]
Yup, `rvm-prompt` returns the correct ruby version while `echo "${rvm_env_string/ruby-}"` is blank.
![rvmrbenv](https://user-images.githubusercontent.com/4336207/39792054-6a8d4222-535c-11e8-8904-fa6f6b71819b.png)

**Comment by pmutch** [comment]
This also affects me

**Comment by dritter** [comment]
@HaoZeke s fix landed in `master` with #1149 . Thanks for the fix @HaoZeke and everybody commenting here.

Closing for now.
